### PR TITLE
Teacher routes

### PR DESCRIPTION
**Pull request by MoeedthePessimist**
Added addQuiz route and quiz schema

### CODE REVIEW

**Review by MoeedthePessimist — COMMENTED** [review]
Looks good so far, merged both admin and teacher routes correctly

File: app.js [hunk 1]
```diff
@@ -15,6 +24,18 @@ var path = require("path");
 var cookieParser = require("cookie-parser");
 var logger = require("morgan");
 var mongoose = require("mongoose");
```
Comment (Th3Un1qu3M4n) [inline app.js:26]: Duplicate delcaration

**Review by Th3Un1qu3M4n — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (MoeedthePessimist) [reply app.js:26]: On it boss

**Review by MoeedthePessimist — COMMENTED** [review]
done and dusted

**Review by Th3Un1qu3M4n — COMMENTED** [review]
Nice Work

**Review by Th3Un1qu3M4n — APPROVED** [review]

**Pull request closed without merging**